libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -83,6 +89,57 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("FS_VOLTSRC", 11, AP_BattMonitor_Params, _failsafe_voltage_source, BattMonitor_LowVoltageSource_Raw),
 
+    // @Param: LOW_VOLT
+    // @DisplayName: Battery low voltage
+    // @Description: Battery voltage to trigger a low battery failsafe. Set to 0 to disable. If the battery voltage drops below this voltage continuously for 10 seconds then the vehicle will switch to RTL mode.
```

Comment:
Maybe "this voltage continuously for 10 seconds" to "this voltage for more than 10 seconds "